runestone/parsons/js/parsons.js

```diff
@@ -1472,9 +1492,9 @@ export default class Parsons extends RunestoneBase {
                     }
                 }
 
-                // Create a canvas and set the passed through fontSize and fontFamily in order to later measure text width - Vincent Qiu (September 2020)
+                // Create a canvas and set the passed through fontSize and fontFamily in order to measure text width - Vincent Qiu (September 2020)
                 var fontSize = linesItem[linesIndex].children[0].fontSize;
-                var fontFamily = linesItem[linesIndex].children[0].fontFamily.split(",")[2];
+                var fontFamily = linesItem[linesIndex].children[0].fontFamily;
                 var tempCanvas = document.createElement("canvas");
                 var tempCanvasCtx = tempCanvas.getContext("2d");
                 tempCanvasCtx.font = fontSize + fontFamily;
```

Comment:
There needs to be a space between the fontsize and the fontFamily if they are jammed together it doesn't recognize them and the sizes are still wrong.

Reply:
Oops, I did catch that in my testing code but forgot to copy that change into the commit.